src/views/ManageList.jsx

```diff
@@ -2,29 +2,49 @@ import { addItem } from '../api/firebase';
 import { useState } from 'react';
 import ShareForm from '../components/ShareForm';
 
-export function ManageList({ listPath }) {
+export function ManageList({ listPath, data }) {
 	const [message, setMessage] = useState('');
 	const [userItem, setUserItem] = useState('');
 	// Presetting item duration to 7, as that option starts
 	// selected on page rendering.
 	const [itemDuration, setItemDuration] = useState(7);
 
+	// lowercases name and removes non-alpha characters
+	const normalizeInput = (name) => {
```

Comment:
Nit: move this into a util file and import it in. Since it's a pure function it doesn't need to be defined only inside of a react component.

Reply:
Also if you have time would suggest writing a test for this function 😄